hail/src/main/scala/is/hail/utils/Graph.scala

```diff
@@ -34,16 +34,16 @@ object Graph {
   }
 
   def pyMaximalIndependentSet(edgesIR: IR, nodeTypeStr: String, tieBreaker: Option[String]): IR = {
-    val nodeType = IRParser.parseType(nodeTypeStr)
+    val nodeType = IRParser.parsePType(nodeTypeStr)
```

Comment:
I think this should be parseType and then `PType.canonical` -- this type string is coming from python, which doesnt have a notion of PTypes

Reply:
actually, that can go below